Added mastercard MCC list

SavingFrame opened the pull request:
Source: https://online.ogs.ny.gov/purchase/snt/awardnotes/7900822712rf018MerchantCategoryCodeList.xls

jleclanche commented:
This is really good! Though I'm confused how you're sourcing this from mastercard if it's hosted on the new york government website -- where did you get the link from?

Two minor changes to make on the csv itself:
- Ensure there is a newline at EOL (it's failing lint because of it)
- Please change the first line to `MCC,DESCRIPTION` to be consistent with the other CSVs

Thank you!

SavingFrame commented:
Updated. I added newline at the end of file and changed row description.
I just found xls on Google, added a little magic to get a csv and was done.
<img width="1032" alt="image" src="https://github.com/jleclanche/python-iso18245/assets/99082173/3678af43-016b-49d2-8b55-93830367110d">

jleclanche commented:
Got it. thank you. Let me try to figure out what the actual original source is before landing it then.

jleclanche commented:
FYI - I don't think this is specifically the mastercast list, so that's why I want to figure out where it's from.

The mastercard list is here: https://www.mastercard.us/content/dam/mccom/en-us/documents/rules/quick-reference-booklet-merchant-edition.pdf

You can see for example in your PR, 1740 is `"Masonry, Stonework, Tile Setting, Plastering, Insulation Contractors"`, whereas the official PDF sets it as `"Insulation, Masonry, Plastering, Stonework, and Tile Setting Contractors"`.
